core/tests/org.fusesource.ide.camel.model.service.core.tests.integration/META-INF/MANIFEST.MF

```diff
@@ -14,14 +14,14 @@ Require-Bundle: org.eclipse.core.filesystem,
  org.assertj.core;bundle-version="2.1.0",
  org.junit;bundle-version="4.12.0",
  org.jboss.tools.locus.mockito;bundle-version="1.9.5",
+ org.eclipse.m2e.maven.runtime;bundle-version="1.6.2",
+ org.eclipse.m2e.core;bundle-version="1.7.1",
```

Comment:
looks like only the ordering changed?

Reply:
yes, it is to avoid some Warning in Eclipse (a workaround an Eclipse bug)